caddyhttp/rewrite/to.go

```diff
@@ -18,8 +18,15 @@ func To(fs http.FileSystem, r *http.Request, to string, replacer httpserver.Repl
 
 	// try each rewrite paths
 	t := ""
+	query := ""
 	for _, v := range tos {
-		t = path.Clean(replacer.Replace(v))
+		t = replacer.Replace(v)
+		tparts := strings.Split(t, "?")
```

Comment:
Hmm, well I like the simplicity of your PR as-is, I'll look at this again tomorrow after I've had some sleep and try to think of ways to break it. In the meantime it looks like this is easy enough to test as you've shown, so we can always add those later if they come up.